### PR TITLE
[T4A6][T13-B1] Low Bao Ling Vivian

### PR DESCRIPTION
**Pull request by sunset1215**
Ready for review

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:3]: don't import everything. import only what is necessary.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * 
+ * @author Vivian
+ */
+public class UtilsTest {
+
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#isAnyNull(java.lang.Object[])}.
+     */
+    @Test
+    public void testIsAnyNull_noNullArgsReturnFalse() {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:23]: you have followed the convention for test names, albeit minor error, separate the three parts with an underscore for better readability

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        Object obj2 = new String("String object");
+        assertTrue("Should have at least 1 null args", Utils.isAnyNull(obj1, obj2));
+    }
+
+    /**
+     * Test method for {@link seedu.addressbook.common.Utils#elementsAreUnique(java.util.Collection)}.
+     */
+    @Test
+    public void testElementsAreUnique_hasNonUniqueElementsReturnFalse() {
+        String string1 = new String("This is string 1");
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:44]: going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.

**Comment by okkhoy** [comment]
@sunset1215 
Some comments added. Please ack & close the PR after reading comments.

**Comment by sunset1215** [comment]
Have read the comments. Thanks for the feedback.

**Pull request closed without merging**